integration/sql/impl/src/visitor.rs

```diff
@@ -52,18 +53,19 @@ impl Visit for TableFactor {
                 context.add_input(name.clone().0);
                 Ok(())
             }
-            TableFactor::Pivot {
-                name, pivot_alias, ..
-            } => {
-                let table = DbTableMeta::new(
-                    name.clone().0,
-                    context.dialect(),
-                    context.default_schema().clone(),
-                );
-                if let Some(pivot_alias) = pivot_alias {
-                    context.add_table_alias(table, vec![pivot_alias.clone().name]);
+            TableFactor::Pivot { table, alias, .. } => {
```

Comment:
would love to see the test verifying that Pivot gets properly translated into `TableLineage`

Reply:
We have test for `Pivot`: `parse_pivot_table` that you yourself added :)

Reply:
you're right. this seems just to be a code refactor.